st2common/st2common/util/monkey_patch.py

```diff
@@ -117,4 +135,8 @@ def is_use_debugger_flag_provided():
         if arg.startswith(PARENT_ARGS_FLAG) and USE_DEBUGGER_FLAG in arg:
             return True
 
+    # 3. Check for ST2_PYCHARM_DEBUG env var
+    if os.environ.get("ST2_PYCHARM_DEBUG", False):
+        return True
```

Comment:
This part is not pycharm specific - it allows enabling debug mode via env var instead of changing the cli args.

```suggestion
    # 3. Check for ST2_DEBUG env var
    if os.environ.get("ST2_DEBUG", False):
        return True
```